README.md

```diff
@@ -40,6 +40,10 @@ Linters.
 
 - [Live Demo Link](https://megagig.github.io/Portfolio-Microverse/)
 
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://megagig.github.io/Portfolio-Microverse/)
+
```

Comment:
- [ ] [OPTIONAL] I think we can remove one Live demo section in order to make sure our Readme is professional as currently there are two of them please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230899601-78a917c0-7412-4185-a576-5bf8150599f4.png)